index.js

```diff
@@ -52,10 +61,54 @@ var osmStream = (function osmMinutely() {
             changeset: +x.getAttribute('changeset'),
             uid: +x.getAttribute('uid'),
             user: x.getAttribute('user'),
-            visible: x.getAttribute('visible') !== 'false'
+            visible: x.getAttribute('visible') !== 'false',
+            tags: parseTags(x)
         };
     }
 
+    function parseBounds(x) {
+        var bounds = get(x, ['bounds']);
+        return [
+            +bounds.getAttribute('maxlat'),
+            +bounds.getAttribute('maxlon'),
+            +bounds.getAttribute('minlat'),
+            +bounds.getAttribute('minlon')
+        ];
+    }
+
+    function parseMembers(x) {
+        var mbrs = x.getElementsByTagName('members');
+        var members = [];
+
+        for (var i = 0; i < mbrs.length; i++) {
+            var mbr = {
+                type: mbrs[i].getAttribute('type'),
+                ref: +mbrs[i].getAttribute('ref'),
+                role: mbrs[i].getAttribute('role')
+            };
+
+            if
```

Comment:
Yep, not quite done yet. I pushed this up in a not-working-state because it looked a lot like [@batpad's code here](https://github.com/batpad/gl-changeset/blob/master/js/overpassToGeoJSON.js).